pomidor.el

```diff
@@ -207,6 +207,9 @@ To disable sounds, set to nil."
 (defvar pomidor--current-history-session nil
   "Hold the current visible pomidor history snapshot.")
 
+(defvar pomidor--system-on-hold? nil
```

Comment:
I believe elisp convention for booleans is `pomidor--system-on-hold-p`

Reply:
Great! Hard to tell when I am at elisp or clojure rsrs. thanks for catching this one.